regolith/helpers/a_proposalhelper.py

```diff
@@ -0,0 +1,190 @@
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "proposals"
+
+def subparser(subpi):
+    # Do not delete --database arg
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("name", help="A short but unique name for the proposal",
+                       )
+    subpi.add_argument("amount", help="value of award",
+                       )
+    subpi.add_argument("title", help="Actual title of the proposal"
+                       )
+    subpi.add_argument("--begin_date", help="The begin date for the proposed grant "
+                                          "in format YYYY-MM-DD.",
+                       default = 'tbd'
+                       )
+    subpi.add_argument("-d", "--duration", help="Duration of proposal in months",
+                       default = 'tbd'
+                       )
+    subpi.add_argument("--due_date", help="The due date for the proposal in format YYYY-MM-DD.",
+                       default = 'tbd'
+                       )
+    subpi.add_argument("-a", "--authors", nargs="+",
+                       help="Other investigator names", default = []
+                       )
+    subpi.add_argument("-c", "--currency",
+                       help="Currency in which amount is specified. Defaults to USD",
+                       default = 'USD'
+                       )
+    subpi.add_argument("-p", "--pi",
+                       help="ID of principal investigator. Defaults to"
+                            "group pi id in regolithrc.json"
+                       )
+    subpi.add_argument("--cppflag", help="Current and pending form (True or False). Defaults to True)",
+                       default = True
+                       )
+    subpi.add_argument("--other_agencies", help="Other agencies to which the proposal has been "
+                                                "submitte. Defaults to False", default = False
+                       )
+    subpi.add_argument("-i", "--institution", nargs="+",
+                       help="The institution where the work will primarily"
+                             "be carried out", default = []
+                       )
+    subpi.add_argument("--months_academic", help="Number of working months in the academic year"
+                                                 "to be stated on the current and pending form."
+                                                 "Defaults to 0",
+                       default = 0
+                       )
+    subpi.add_argument("--months_summer", help="Number of working months in the summer to be"
+                                                "stated on the current and pending form"
```

Comment:
I think you will have to work on whitespace in all your help messages.  Run the code locally if you can (I can send a test proposal file if you need) to check for these things (pytest won't show this up)

Reply:
My bad! Found this error in multiple places by running `regolith helper a_proposal` again and reading the help carefully